kubernetes/client/models/v1beta1_custom_resource_definition_status.py

```diff
@@ -101,8 +101,6 @@ def conditions(self, conditions):
         :param conditions: The conditions of this V1beta1CustomResourceDefinitionStatus.
         :type: list[V1beta1CustomResourceDefinitionCondition]
         """
-        if conditions is None:
-            raise ValueError("Invalid value for `conditions`, must not be `None`")
```

Comment:
this is generated, and will be overwritten in next release, right?

Reply:
yes, the proper fix will come in 1.16 https://github.com/kubernetes/kubernetes/pull/64996

somehow in the test, the response is deserialized into a dict instead of a CRD object, which causes the attribute assertion to fail. Perhaps this workaround isn't ideal either. I will abandon this pull and verify the CR patch methods manually in the next couple of releases

/close